mkdocs.yml

```diff
@@ -45,67 +45,101 @@ plugins:
-        - 'Project Setup': 'getting-started/example-deployment/project-setup.md'
-        - 'Writing the Schema': 'getting-started/example-deployment/writing-the-schema.md'
-        - 'Running the Deployment': 'getting-started/example-deployment/running-the-deployment.md'
-        - 'Using the API': 'getting-started/example-deployment/using-the-api.md'
-      - 'Advanced User Guide':
-        - 'Aggregation': 'getting-started/advanced-guide/aggregation.md'
-        - 'Import and Export':
-          - 'Using CSV': 'getting-started/advanced-guide/import-export/csv.md'
-        - 'Cardinality': 'getting-started/advanced-guide/cardinality.md'
-    - 'Gaffer 2.0':
-      - Changelist: 'gaffer2.0/changelist.md'
-      - Deprecations: 'gaffer2.0/deprecations.md'
-      - Dependencies: 'gaffer2.0/dependencies.md'
-      - 'Accumulo Migration': 'gaffer2.0/accumulo-migration.md'
-      - 'Federation Changes': 'gaffer2.0/federation-changes.md'
-      - 'Accumulo Kerberos': 'gaffer2.0/accumulo-kerberos.md'
-      - 'Log4j in Gaffer': 'gaffer2.0/log4j.md'
-    - 'Developer Docs':
-      - 'Development': 'dev/development.md'
-      - 'Remote Coding Environments': 'dev/remote-coding-environments.md'
-      - 'Managing Dependencies': 'dev/managing-dependencies/managing-dependencies.md'
-      - 'Ways of Working': 'dev/ways-of-working.md'
-      - 'Sketches custom deserialisation': 'dev/rest-api-sketches.md'
-      - 'Docker': 'dev/docker.md'
-      - Kubernetes:
-        - 'Kubernetes Guide': 'dev/kubernetes-guide/kubernetes.md'
-        - 'Deploy an Empty Graph': 'dev/kubernetes-guide/deploy-empty-graph.md'
-        - 'Add your schema': 'dev/kubernetes-guide/deploy-schema.md'
-        - 'Change the Graph Id and Description': 'dev/kubernetes-guide/change-graph-metadata.md'
-        - 'Adding your own libraries and functions': 'dev/kubernetes-guide/add-libraries.md'
-        - 'Changing Accumulo Passwords': 'dev/kubernetes-guide/change-accumulo-passwords.md'
-      - Components:
-        - 'Components/Maven Modules': 'dev/components/components.md'
-        - 'Cache': 'dev/components/cache.md'
-        - 'Data': 'dev/components/data.md'
-        - 'Graph': 'dev/components/graph.md'
-        - 'Store': 'dev/components/store.md'
-        - 'Operation': 'dev/components/operation.md'
-        - 'Serialisation': 'dev/components/serialisation.md'
-        - 'Integration Tests': 'dev/components/integration-test.md'
-        - 'Core REST': 'dev/components/core-rest.md'
-        - 'Spring REST': 'dev/components/spring-rest.md'
-        - 'Accumulo Store': 'dev/components/accumulo-store.md'
-        - Libraries:
-          - dev/components/libraries/bitmap.md
-          - dev/components/libraries/flink.md
-          - dev/components/libraries/sketches.md
-          - dev/components/libraries/spark.md
-          - dev/components/libraries/time.md
-    - 'Reference':
+        - 'Project Setup': 'development-guide/example-deployment/project-setup.md' 
```

Comment:
Think the example deployment is maybe more relevant to the admin guide section? Its kind of an example of how an admin could set up an instance. A dev can usually get by just spinning up a simple map store instance to test any changes